modules/claims_api/lib/bd/bd.rb

```diff
@@ -172,7 +175,7 @@ def client
 
     def build_body(options = {})
       data = {
-        systemName: options.fetch(:system_name, 'VA.gov'),
+        systemName: options[:system_name].presence || 'VA.gov',
```

Comment:
👍 - do we also want to fix `trackedItemIds` below?

Reply:
I think we'd want to. This could change our previous `[]` to a `nil`

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/17763